relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -600,6 +611,41 @@ fn extract_span_metrics(
     Ok(())
 }
 
+static SQL_TABLE_EXTRACTOR_REGEX: Lazy<Regex> =
+    Lazy::new(|| Regex::new(r#"(?i)from(\s|\()+(?P<table>(\w+(\.\w+)*))(\s|\))+"#).unwrap());
```

Comment:
Should we also consider other sql statements like INSERT? That is, `(from|into)`?

Reply:
Does this match formatted multiline queries, for example

```sql
SELECT
    * 
FROM
    table_name
WHERE
    1
```